website/blog/2022-08-01-aluminium.md

```diff
@@ -0,0 +1,355 @@
+---
+author: Rikito Taniguchi
+title: Metals v0.11.8 - Aluminium
+authorURL: https://twitter.com/<login>
+authorImageURL: https://github.com/<login>.png
+---
+
+We're happy to announce the release of Metals v0.11.8, which brings a number of improvements for both Scala 2 and Scala3.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">78</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">74</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">14</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center"></td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">6</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/<num>?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs and
+Sublime Text. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- [Scala 3] Type completion
+- [Scala 3] Auto complete and import extension methods
+- [Scala 3] ConvertToNamedParameters code action
+- [Scala 3] ScaladocCompletion for Scala3
+- [Scala 3] Add interpolation completions
+- [Scala 2] Automatically import types in string interpolations
+- More stable code navigation across subprojects
+- Code Action documents
+- Support Scala 3.2.0-RC3, Scala 3.2.0-RC2
+
+and a lot of bugfixes!
+
+## [Scala 3] Type Competion
+
+## [Scala 3] Auto complete and import extension methods
+
+Scala3 introduced `extension methods` that allows defining new methods to the existing types.
+
+However, previously, Metals couldn't auto-complete extension methods, and developers have to find an appropriate extension methods from workspace and import it by hand. It is quite time-consuming.
+
+Now, Metals provides auto-completion for the extension methods and auto-import them!
+
+## [Scala 3] ConvertToNamedParameters code action
+
+[Metals 0.11.7 added `ConvertToNamedParameters` code action to Scala2](https://scalameta.org/metals/blog/2022/07/04/aluminium#scala-2-add-converttonamedarguments-code-action).
+
+Thanks to the contribution by [@jkciesluk](https://github.com/jkciesluk), this feature is now available for Scala3!
+
+![convert-to-named](https://i.imgur.com/9i7MWoQ.gif)
+
+## [Scala 3] Scaladoc completion
+
+Metals now supports auto completing scaladoc completion in Scala3. Typing `/**`, you can auto-complete scaladoc template for the method, class, and etc!
+
+![scala-doc-completion](https://i.imgur.com/MEJUXr3.gif)
+
+## [Scala 3] Interpolation completion
+
+## [Scala 2] Auto import types in string interpolations
+
+## More stable code navigation across subprojects
```

Comment:
I'd like to mention something is improved by https://github.com/scalameta/metals/pull/4176
Do you think this is enough, or do you think we don't need to mention it? @kpodsiad

Reply:
Yeah, providing concrete examples is always a good idea! 
However, it might be challenging to describe this change to end-users, so I'm also fine with just telling them something has improved in this version without a screenshot.